features/canvas-2d-context-willreadfrequently.yml

```diff
@@ -0,0 +1,6 @@
+name: willReadFrequently
```

Comment:
@ddbeck - for consistency with naming from above does how about `canvas-2d-willreadfrequently.yml`?